### DISCUSSION TITLE
Well my Raspberry Pi 4 died...

### DISCUSSION THREAD

**Discussion by JasonLG1979** [dtcooper/raspotify#521] [General]
As the title suggests my Pi 4 has bit the dust. It's a sad day especially considering that a replacement is going to be basically impossible to get my hands on. I have however ordered a super cheap x86_64 mini pc that will stand-in for the Pi so all hope is not lost. My plan is to run Debian 11 (Bullseye) on it so I don't see it affecting Raspotify a great deal, this is just a heads up that I can no longer give super specific advice or support for Pi hardware, nor will I be able to debug ARM specific bugs if they should arise.

**Comment by dtcooper** [comment]
> It's a sad day especially considering that a replacement is going to be basically impossible to get my hands on.

How come? Can I help you get another one? 😛

**Reply by JasonLG1979** [reply]
It's not a matter of money. It's a matter of availability. I'll pick one up as soon as I can. The bright side is that this is a good excuse to my wife as to why I need another "real" computer. My Pi was also my Plex and file/backup server.

**Reply by JasonLG1979** [reply]
There are no brick and mortar Pi retailers near me so I'm at the mercy of ordering online and currently the only Pi's I can find are either at scalper prices or they ship from Europe so shipping is crazy.

**Reply by JasonLG1979** [reply]
I've also managed to convince her that I also need a new USB DAC (Topping D10s) since the audio out on a mini-pc is going to be garbage.

**Reply by dtcooper** [reply]
I guess I can't help with a marital dispute... ¯\\\_(ツ)\_/¯

**Reply by JasonLG1979** [reply]
There's no dispute. It's just that my wife is fairly tech savvy and she controls the finances because I have no impulse control when it comes to money. So it takes something to break before I get an upgrade and I have to plead my case. Again that's a good thing. I'd buy magic beans...

**Reply by JasonLG1979** [reply]
I've got my eye on this little guy:

https://www.amazon.com/gp/product/B09KH7MCD1

It's no supercomputer by any means but it should run circles around a Pi and more than suit my needs. Again though when Pi's become more available I plan on getting one and the mini-pc will be my main home server and the Pi will be my **Raspoti-Box™**.

Windows (:vomiting_face:) ofc will be going bye-bye.

**Comment by JasonLG1979** [comment]
Update: I have managed to order a Pi Zero v2 from [The PiHut](https://thepihut.com).

Going forward the Pi Zero v2 will be our "Potato", Raspotify's lowest spec reference device. That's not to say that Raspotify won't continue to run on a PI 2, it's just that I don't own a 2 (and why and where would I buy one?).

If Raspotify works on a Pi Zero v2 it should work just fine on a Pi 2 - 4.
